homeassistant/components/hassio/config_flow.py

```diff
@@ -19,6 +19,6 @@ async def async_step_system(
     ) -> ConfigFlowResult:
         """Handle the initial step."""
         # We only need one Hass.io config entry
-        await self.async_set_unique_id(DOMAIN)
-        self._abort_if_unique_id_configured()
+        if self._async_current_entries():
```

Comment:
Have you checked that we don't rely on the unique_id anywhere in the integration?

Reply:
I don't think we use it, but please check.

Reply:
I only found one reference to an config entry id
https://github.com/home-assistant/core/blob/576aaec75a98fd0a2887cdcc8c23ec6dea9cfcfa/homeassistant/components/hassio/discovery.py#L167

will check this in detail later

Reply:
That's referring to other config entries that are discovered via HASSIO discovery. So that's fine. :+1: